integration-cli/docker_cli_daemon_test.go

```diff
@@ -2848,3 +2849,36 @@ func (s *DockerDaemonSuite) TestExecWithUserAfterLiveRestore(c *check.C) {
 	out, err = s.d.Cmd("stop", "top")
 	c.Assert(err, check.IsNil, check.Commentf("Output: %s", out))
 }
+
+func (s *DockerDaemonSuite) TestRemoveContainerAfterLiveRestore(c *check.C) {
+	testRequires(c, DaemonIsLinux, IsOverlay)
+	s.d.StartWithBusybox(c, "--live-restore", "-s", "overlay")
```

Comment:
I think you need to use `--storage-driver`. https://github.com/docker/docker/blob/bf6eb855391e166a65db284567686fafbe4a9a0a/integration-cli/daemon/daemon.go#L243-L245